src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -45,6 +48,12 @@ public DefaultServer(List<Type> typesToInclude)
 
             configurationBuilderCustomization(configuration);
 
+            if (!EnableDiagnostics)
```

Comment:
If the new field was `DisableDiagnostics` instead, then this logic would a bit easier to follow, since it would be  "if disabled, then do action to disable".